docs/src/app/(public)/careers/design-engineer/page.tsx

```diff
@@ -0,0 +1,221 @@
+            pros and cons of API design choices across headless UI libraries.
+          </li>
+          <li className="mb-0.5">
+            Expert knowledge of a11y, including deep familiarity with ARIA
+            guidelines, WCAG success criterion, and screen reader technologies.
+          </li>
+          <li className="mb-0.5">Advanced understanding of design principles.</li>
+          <li className="mb-0.5">
+            A passion for craft, a keen eye for detail, and exquisite taste.
+          </li>
+          <li className="mb-0.5">
+            Excellent written and verbal communication skills.
+          </li>
+          <li className="mb-0.5">
+            Experience working remotely and communicating asynchronously.
+          </li>
+        </ul>
+        <h2 className="mt-10 mb-4 scroll-mt-6 text-xl font-medium text-balance">
+          Benefits
+        </h2>
+        <ul className="mb-4 ml-4.5 list-disc">
+          <li className="mb-0.5">$200k–$290k USD base salary.</li>
+          <li className="mb-0.5">
+            Subsidized healthcare package (dependent on employment path and
+            location).
+          </li>
+          <li className="mb-0.5">
+            Flexible time-off. We provide 33 days of paid time-off globally.
+          </li>
+          <li className="mb-0.5">
+            100% remote. Our entire company is globally distributed.
+          </li>
+          <li className="mb-0.5">
+            Company retreats. We meet up every 8 months for a week of work and fun.
+          </li>
+          <li className="mb-0.5">
+            Office equipment. We let you choose the hardware of your choice.
+          </li>
+          <li className="mb-0.5">
+            20% time. Allocate 20% of your time towards personal and professional
+            development.
+          </li>
+          <li className="mb-0.5">
+            Education budget. We provide mentorship and send you to events that help
+            you build your network and skills.
+          </li>
+        </ul>
+        <p className="mb-4">
+          The actual salary will be determined by skill-level, experience, and
+          location. More information about the specific compensation package will be
+          shared during the hiring process.
+        </p>
+        <h2 className="mt-10 mb-4 scroll-mt-6 text-xl font-medium text-balance">
+          Application
+        </h2>
+        <p className="mb-4">The hiring process will consist of 5 stages:</p>
+        <ol className="mb-4 ml-4.5 list-inside list-decimal">
+          <li className="mb-0.5">Resume review.</li>
+          <li className="mb-0.5">React challenge (asynchronous).</li>
+          <li className="mb-0.5">60-minute meeting with Colm (Product lead).</li>
+          <li className="mb-0.5">
+            60-minute meeting with Marija (Engineering Manager).
+          </li>
+          <li className="mb-0.5">90-minute meeting with the Base UI team.</li>
+        </ol>
```

Comment:
```suggestion
        <p className="mb-4">The hiring process will consist of 4 stages:</p>
        <ol className="mb-4 ml-4.5 list-inside list-decimal">
          <li className="mb-0.5">30-minute meeting with Colm (Product lead)</li>
          <li className="mb-0.5">React challenge (asynchronous).</li>
          <li className="mb-0.5">90-minute meeting with the Base UI team.</li>
          <li className="mb-0.5">60-minute meeting with the CEO (Cultural fit interview).</li>
        </ol>
```

I will join the Team interview, it's anyway a bit longer, I can ask questions I'd like to clarify on, or alternatively I can join the first call and you can join the team call, up to you, I think both of them can work.